src/components/Registration/Registration.jsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+import { RegistrationCards } from "./Registration.config";
+import styles from "./Registration.module.sass";
+import Link from "next/link";
+
+const Registration = () => {
+  return (
+    <>
+      <div className={styles.reg_header} key="reg-header">
+        Logo
+      </div>
+      <div className={styles.reg_body} key="reg-body">
+        <span class="material-symbols-outlined">
+          <link
```

Comment:
I'm not sure what does that link tag means ???

Reply:
I tried using Google symbols, there is a span and a link element like here - https://fonts.google.com/icons?selected=Material+Symbols+Outlined:group:FILL@0;wght@400;GRAD@0;opsz@24. It seems that the link element have to appear somewhere in code (maybe in another place) otherwise icons don't work at all.      

I don't know how to correctly insert icons and change their sizes etc. There is some much information, I am getting lost.